assets/css/less/wpbdp.less

```diff
@@ -955,57 +955,16 @@ a.wpbdp-button {
     }
 }
 
-.wpbdp-wp-theme-twentysixteen {
-    .wpbdp-listing .social-fields {
-        .social-field.linkedin {
-            vertical-align: text-top;
-        }
-    }
-}
-
-.wpbdp-wp-theme-twentyseventeen {
-
-    .single-featured-image-header {
-        display: none;
-    }
-
-    .wpbdp-listing {
-        
-        .social-fields .twitter.twitter-handle {
-            vertical-align: text-top;
-            padding-top: 1px;
-        }
-    }
-}
-
-.wpbdp-wp-theme-twentynineteen {
-
-    .wpbdp-listing .social-fields {
-        .twitter.twitter-handle {
-            vertical-align: text-top;
-        }
+.wpbdp-thumbnail {
+    max-width: var(--bd-thumbnail-width);
```

Comment:
Let's add a fallback in here for each of these lines so it'll look decent in older browsers too.
max-width: 200px;
max-width: var(--bd-thumbnail-width);

Reply:
Done :)